tests/RobotFramework/tests/tedge/device_id/device_id_usecases.robot

```diff
@@ -0,0 +1,69 @@
+*** Settings ***
+Documentation       This suite covers all usecases in the issue #3369
+
+Resource            ../../../resources/common.resource
+Library             Cumulocity
+Library             ThinEdgeIO
+Library             ../../.venv/lib/python3.11/site-packages/robot/libraries/String.py
```

Comment:
This looks to be an accidental import. Check if the `String` RobotFramework is required, if so, then change to:

```suggestion
Library             String
```

Reply:
Thanks for catching this. IDE added it automatically, it was not intended and safe to remove.